xdsl/dialects/vector.py

```diff
@@ -179,22 +154,24 @@ def get(
 @irdl_op_definition
 class MaskedloadOp(IRDLOperation):
```

Comment:
Maybe good opportunity to rename this to `MaskedLoadOp`  à la MLIR too? Same for the masked store?
if it's a fuss, no biggie.